src/test/java/com/jitterted/mobreg/adapter/in/web/member/MemberControllerTest.java

```diff
@@ -91,6 +90,38 @@ public void memberDeclinesWillBeDeclinedForEnsemble() throws Exception {
                 .isTrue();
     }
 
+    @Test
+    public void ensembleFormShowsMembersWhoAcceptedForEnsemble() throws Exception {
```

Comment:
Is this test verifying anything that the EnsembleSummaryViewTest isn't? I generally don't test the contents of Views in the Model in a Controller test, because it's awkward (a lot of setup needed, as you found). I assume that if the ESView is in the Model, that's good enough.

Reply:
Right, I was also doubting the same thing. Agreed, will get rid.